src/ipfs-service.js

```diff
@@ -17,14 +14,13 @@ const get = async (cid, { ipfsGateway = DEFAULT_IPFS_GATEWAY } = {}) => {
     throw Error(`Failed to load content from ${publicUrl}`);
   }
   const arrayBuffer = await res.arrayBuffer();
-  return Buffer.from(arrayBuffer);
+  return new Uint8Array(arrayBuffer);
 };
 
 const add = async (content, { ipfsGateway = DEFAULT_IPFS_GATEWAY } = {}) => {
-  console.log('add', await kuboRpcPromise);
-  const { create } = await kuboRpcPromise;
-  const ipfsClient = create('/dns4/ipfs-upload.iex.ec/https/');
+  const ipfsClient = create('/dns4/ipfs-upload.v8-bellecour.iex.ec/https');
```

Comment:
use a constant in config.js : IPFS_UPLOAD_URL = /dns4/ipfs-upload.v8-bellecour.iex.ec/https